### ISSUE TITLE
First time keyboard navigation focus box is not visible.

### ISSUE THREAD

**Issue by HaoYangMS** [microsoft/fluentui#13020]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: Chrome, Edge

### Describe the issue:
When using keyboard to navigate to the page first time, the focused element do not show the focus box. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/haoyangms/full/GRpQJzQ

#### Repro Steps
1.	Open codepen https://codepen.io/haoyangms/full/GRpQJzQ
2.	Put the focus outside the page (e.g. on browser's address bar)
3.	Press Tab for several time until the focus appears on the “Hello world” button. 
4.	Observe the issue. The page shows “the button was focused” but the focus box does not appear.
5.	If we tab away and shift+tab back, the focus box can be shown properly.

#### Actual behavior:
The focus box does not be shown

#### Expected behavior:
The focus box should be shown when using keyboard navigation.

### Documentation describing expected behavior

[repro step.pptx](https://github.com/microsoft/fluentui/files/4585926/repro.step.pptx)

**Comment by aneeshack4** [comment]
@HaoYangMS Thanks for reaching out to us! At first I wasn't able to repro this: 

![Screen Recording 2020-05-06 at 10 38 32 AM](https://user-images.githubusercontent.com/4161791/81197545-0a55b480-8f86-11ea-86b7-8bd082adb74b.gif)

But later, I went back and tried it again and on the first tab, it didn't show the focus box but when I reverse navigated using back tab, the focus rect showed up again... seems intermittent @khmakoto any ideas as to what's happening here?

**Comment by khmakoto** [comment]
This is really weird, why isn't it working just at the beginning? It also does not repro on old Edge or Firefox.

The thing I do see is that at first the `body` has no class, upon first focusing on the button it's still that way, but once you move focus away from the button the class `ms-Fabric--isFocusVisible` is applied to the body. If you click on the page at any point the class `ms-Fabric--isFocusHidden` is the one applied. I think what's happening is that we expect an original interaction, either a keyboard focus or a mouse click on the page to initialize the focus rectangles. But since we are starting with the focus on the browser instead of on the page the first time we focus on the page is the button itself, so we don't initialize the focus rectangles until just after this interaction, which is why you actually see the border the second time you focus on the button.

@dzearing how does this sound? Does it seem too far-fetched? If this sounds about right, what would you think would be a good way to solve this problem?
